README.md

```diff
@@ -786,6 +786,49 @@ We need to create an HTTP factory by using the `createHttpFactory()` function, p
 - `get()` - A proxy for Angular `TestBed.get()`
 - `expectOne()` - Expect that a single request was made which matches the given URL and it's method, and return its mock request
 
+Spectator also makes testing concurrent requests easier. Similar to `spectator.expectOne()`, you can use,
+`spectator.expectConcurrent()`. Suppose you have the following service that aggregates todos from multiple
+sources.
+
+```ts
+export class TodoDataService {
+    constructor(private http: HttpClient) {}
+
+    collectTodos() {
+        return merge(
+            this.http.get('/api1/todos'),
+            this.http.get('/api2/todos')
+        );
+    }
+}
+```
+
+The test would look like this
+
+```ts
+import { createHttpFactory, HttpMethod } from '@ngneat/spectator';
+import { TodosDataService } from './todos-data.service';
+
+describe('HttpClient testing', () => {
+  let spectator: SpectatorHttp<TodosDataService>;
+  const createHttp = createHttpFactory(TodosDataService);
+  
+  beforeEach(() => spectator = createHttp());
+
+  it('can test current http calls', () => {
+    spectator.service.getTodos().subscribe();
+    const reqs = spectator.expectConcurrent([
+        { url: '/api1/todos', method: HttpMethod.GET },
+        { URL: '/api2/todos', method: HttpMethod.GET }
+    ]);
+
+    reqs.forEach(req => {
```

Comment:
I like this approach, but I think we can make it cleaner. Something like: `flushAll()`

Reply:
Are you talking about exposing another method on SpectatorHttp?

Possibly something like `spectator.flushAll(requests, [ { foo: 'lorem' }, { bar: 'ipsum' }])`?

Reply:
Yes

Reply:
Okay, I'll make those changes.